code-challenges.test.js

```diff
@@ -12,6 +12,13 @@
 
 // --------------------1) Prompt: Create a function that takes in a number (greater than 2) and returns an array containing the Fibonacci sequence. The length of the array is determined by the argument of the function.
 
+const fibonacciSequence = num > 2
+let 
+if (num > 2) {
+ return 
+
+//  I think I'm having a hard time grasping how to do this. I looked uo the fibonacci sequence and understand the concept of it. But I dont get the idea of inputting one number and returning an array; let alone an array that includes the sequence. I'm sure it would need a conditional to decide what to do/ return. But im still not sure how you would return an entire array from one number. 
```

Comment:
Make sure to pay attention during assessment review!
The key concept of this challenge is understanding how to access the previous 2 index values in relation to the current index of iteration.